### ISSUE TITLE
Overfull \hboxes in TOC pramanavarttikavrtti.tex

### ISSUE THREAD

**Issue by ppasedach** [paddymcall/SARIT-pdf-conversions#31]
Adding the `draft` class option to memoir I can see overfull hboxes in the TOC, wherever the page number is more than three digits. This is possibly a problem created by the devanagari numbering.

**Comment by ppasedach** [comment]
```
\renewcommand*{\cftdotsep}{1}
\setpnumwidth{3em}
\setrmarg{4em}
```

Or something similar in the preamble will fix it. See http://tex.stackexchange.com/questions/49887/overfull-hbox-warning-for-toc-entries-when-using-memoir-documentclass .

**Comment by paddymcall** [comment]
Fixed in paddymcall/Stylesheets/commit/5801e28f9d16a0aecc1158c61feb52db0616258c
